### PR TITLE
Use MusicStore gating changes in CoreClr

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
CC @victorhurdugaci, @pranavkm

### CODE REVIEW

File: KoreBuild-dotnet/build/KoreBuild.sh [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
+version=$(<nix.cli.version)
```
Comment (victorhurdugaci) [inline KoreBuild-dotnet/build/KoreBuild.sh]: Do you need a different mac/linux version?

File: KoreBuild-dotnet/build/KoreBuild.cmd [hunk 2]
```diff
@@ -14,7 +14,7 @@ IF "%KOREBUILD_DOTNET_CHANNEL%"=="" (
 )
 
 IF "%KOREBUILD_DOTNET_VERSION%"=="" (
-    SET KOREBUILD_DOTNET_VERSION=1.0.0.001496
+    SET /P KOREBUILD_DOTNET_VERSION=<win.cli.version
```
Comment (victorhurdugaci) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: That file doesn't exist yet?

Comment (victorhurdugaci) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: A little picky but you should call the file `cli.version.<os>` to keep all files together when you sort by name

Comment (ryanbrandenburg) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: Not yet, the CoreClr build will have to pass the first time before we can merge this.

Comment (victorhurdugaci) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: Oh, ok. I guess you could also manually create that file to make this work

Comment (ryanbrandenburg) [inline KoreBuild-dotnet/build/KoreBuild.cmd]: Will do #2 on my next pass.

**Comment by ryanbrandenburg** [comment]
:up: :date:

File: KoreBuild-dotnet/build/KoreBuild.sh [hunk 3]
```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env bash
 
+if [ $(uname) == Darwin ]; then
```
Comment (victorhurdugaci) [inline KoreBuild-dotnet/build/KoreBuild.sh]: test uname https://github.com/aspnet/Universe/blob/d679c85aa386f00afde6ce7062f286dedb76c7f9/build.sh#L3

**Comment by victorhurdugaci** [comment]
Looks good :shipit:

**Comment by victorhurdugaci** [comment]
Btw, please make this change in https://github.com/aspnet/KoreBuild too once you merge

**Comment by ryanbrandenburg** [comment]
Just to be clear for posterity, what we talked about in person was making this change to KoreBuild _instead_ of, rather than in addition to, making it here.

**Comment by victorhurdugaci** [comment]
Correct. I have PR that will completely remove KoreBuild from universe https://github.com/aspnet/Universe/pull/410

File: KoreBuild-dotnet/build/KoreBuild.sh [hunk 4]
```diff
@@ -1,7 +1,13 @@
...
 
+if test `uname` = Darwin; then
+    version=$(<cli.version.darwin)
+else
+    version=$(<cli.version.unix)
+fi
+
 [ -z "$KOREBUILD_DOTNET_CHANNEL" ] && KOREBUILD_DOTNET_CHANNEL=beta
-[ -z "$KOREBUILD_DOTNET_VERSION" ] && KOREBUILD_DOTNET_VERSION=1.0.0.001496
+[ -z "$KOREBUILD_DOTNET_VERSION" ] && KOREBUILD_DOTNET_VERSION=version
```
Comment (victorhurdugaci) [inline KoreBuild-dotnet/build/KoreBuild.sh]: This will have to read the correct version based on OS: Linux/OSX

Comment (ryanbrandenburg) [inline KoreBuild-dotnet/build/KoreBuild.sh]: Isn't that take care of above, or are you talking about something else?

Comment (victorhurdugaci) [inline KoreBuild-dotnet/build/KoreBuild.sh]: Ha, I missed that

Comment (victorhurdugaci) [inline KoreBuild-dotnet/build/KoreBuild.sh]: Btw, shouldn't that be $version?

**Comment by ryanbrandenburg** [comment]
Closing this since it's replaced by aspnet/KoreBuild/pull/12.

**Pull request closed without merging**